CONTRIBUTING.md

```diff
@@ -30,3 +30,61 @@ tests that validate implemented features and the presence or lack of defects.
 Additionally, the code should follow any stylistic and architectural guidelines 
 prescribed by the project. In the absence of such guidelines, mimic the styles
 and patterns in the existing code-base.
+
+## Browser support
+
+We configure [Autoprefixer](#autoprefixer) to support the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
+- Internet Explorer 8
```

Comment:
Project has reference to IE7 https://github.com/cfpb/college-costs/blob/master/gulp/tasks/styles.js#L40

Reply:
@anselmbradford this has been addressed.